blocks/footer/footer.css

```diff
@@ -1,14 +1,312 @@
-footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
+.footer-wrapper {
+  background: var(--background-color-light-grey);
+}
+
+.footer.block .section.footer-top-note {
+  padding-top: 0;
+  padding-bottom: 1.25rem;
+}
+
+.footer.block .footer-disclaimer .section-container {
+  padding-left: 0.9375rem;
+  padding-right: 0.9375rem;
+  padding-bottom: 1.875rem;
+}
+
+.footer.block .footer-links .section-container {
+  padding-left: 1.75rem;
+  padding-right: 2rem;
+  padding-bottom: 3.438rem;
+  border-bottom: 0.0625rem solid var(--horizontal-footer-dash-gray);
+}
+
+@media (width > 48rem){
```

Comment:
Nit: Please move media queries to the end.

Reply:
Also queries should be with `>=` as done here https://github.com/aemsites/icicidirect/blob/main/styles/styles.css#L402

Reply:
done